Resources/Prototypes/_Sunrise/Catalog/uplink_catalog.yml

```
@@ -459,6 +458,56 @@
       - NukeOpsUplink
       - LoneOpsUplink
 
+- type: listing
+  id: UplinkMagazineBoxHeavyRifle
+  icon: { sprite: Objects/Weapons/Guns/Ammunition/Boxes/light_rifle.rsi, state: base }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)